pkg/virt-handler/vm.go

```diff
@@ -291,6 +292,12 @@ func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, shouldDeleteVm bool) erro
 		return err
 	}
 
+	// Map Container Registry Disks to block devices Libvirt can consume
+	vm, err = registrydisk.MapRegistryDisks(vm)
+	if err != nil {
+		return err
+	}
```

Comment:
This is an example of what I want us to avoid doing - it is hardcoding interaction with a specific disk technology. I would like to see this using the REST api approach I described, where it queryies the container at a designated end point to get the desired kubvirt.api.v1.DiskSource back in a generic manner.